src/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -585,7 +585,7 @@ static function (Task $task) {
         throw $this->createAccessDeniedHttpException();
     }
 
-    private function injectValuesForCustomLayout(array &$layout): void
+    private function injectValuesForCustomLayout(?array &$layout): void
     {
         foreach ($layout['children'] as &$child) {
```

Comment:
When $layout is null this will introduce a warning, right?
```suggestion
        foreach ($layout['children'] ?? [] as &$child) {
```